Editor/AGS.Editor.Tests/Panes/FreePanControlTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Windows.Forms;
+using NUnit.Framework;
+
+namespace AGS.Editor
+{
+    [TestFixture]
+    public class FreePanControlTests
+    {
+        private FreePanControlTestableWrapper _pane;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _pane = new FreePanControlTestableWrapper();
+        }
+
+        [Test]
+        public void IsPanningDefaultsToFalse()
+        {
+            Assert.That(_pane.IsPanning, Is.False);
+        }
+
+        [Test]
+        public void IsPanningIsTrueOnMiddleMouseClick()
+        {
+            _pane.OnMouseDown(new MouseEventArgs(MouseButtons.Middle, 1, 5, 5, 0));
+            Assert.That(_pane.IsPanning, Is.True);
+        }
+
+        [Test]
+        public void PanGrabbedInvokesOnMiddleMouseClick()
+        {
+            bool panGrabbedInvoked = false;
+            _pane.PanGrabbed += (s, e) => panGrabbedInvoked = true;
+            _pane.OnMouseDown(new MouseEventArgs(MouseButtons.Middle, 1, 5, 5, 0));
+            Assert.That(panGrabbedInvoked, Is.True);
+        }
+
+        [Test]
+        public void IsPanningReturnsToFalseOnPan()
+        {
+            _pane.OnMouseDown(new MouseEventArgs(MouseButtons.Middle, 1, 5, 5, 0));
+            Assert.That(_pane.IsPanning, Is.True);
+            _pane.OnMouseMove(new MouseEventArgs(MouseButtons.Middle, 1, 4, 4, 0));
+            Assert.That(_pane.IsPanning, Is.True);
+            _pane.OnMouseUp(new MouseEventArgs(MouseButtons.Middle, 1, 3, 3, 0));
+            Assert.That(_pane.IsPanning, Is.False);
+        }
+
+        [Test]
+        public void CursorIsModifiedAndReturned()
+        {
+            _pane.Cursor = Cursors.IBeam;
+            Assert.That(_pane.Cursor != _pane.GetPanCursor(), Is.True);
```

Comment:
When comparing 2 values do this instead. Same for every other test
```suggestion
            Assert.That(_pane.Cursor Is.EqualTo(_pane.GetPanCursor());
```

Reply:
I fixed this in the last commit.